src/www/ui/api/Controllers/FileInfoController.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/*
+ SPDX-FileCopyrightText: © 2018 Siemens AG
+ Author:  Author: Soham Banerjee <sohambanerjee4abc@hotmail.com>
+ SPDX-FileCopyrightText: © 2023 Soham Banerjee <sohambanerjee4abc@hotmail.com>
+ SPDX-License-Identifier: GPL-2.0-only
+*/
+
+/**
+ * @file
+ * @brief Controller for copyright queries
+ */
+
+namespace Fossology\UI\Api\Controllers;
+
+use Fossology\Lib\Auth\Auth;
+use Fossology\Lib\Dao\UploadDao;
+use Fossology\Lib\Db\DbManager;
+use Fossology\UI\Api\Helper\ResponseHelper;
+use Fossology\UI\Api\Models\FileInfo;
+use Fossology\UI\Api\Models\Info;
+use Fossology\UI\Api\Models\InfoType;
+use Psr\Http\Message\ServerRequestInterface;
+
+
+class FileInfoController extends RestController
+{
+  /**
+   * @var ContainerInterface $container
+   * Slim container
+   */
+  protected $container;
+
+  /**
+   * @var viewInfo $viewInfo
+   * View Info object
+   */
+  private $viewInfo;
+
+  public function __construct($container)
+  {
+    parent::__construct($container);
+    $this->container = $container;
```

Comment:
Parent class initializer does this for you. No need to override it. Please remove the line.

https://github.com/fossology/fossology/blob/3d6195aeff303a01511b5b7662e138f059557f16/src/www/ui/api/Controllers/RestController.php#L52